src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -1228,9 +1228,9 @@ public void run() {
         GT_Values.RA.addCentrifugeRecipe(GT_ModHandler.getModItem("GalaxySpace", "item.GlowstoneDusts", 1L, 2), GT_Values.NI, GT_Values.NF, GT_Values.NF, new ItemStack(Items.glowstone_dust, 2, 0), CustomItemList.EnceladusStoneDust.get(1L), CustomItemList.EnceladusStoneDust.get(1L), GT_Values.NI, GT_Values.NI, GT_Values.NI, new int[]{10000, 9000, 7500}, 300, 120);
         GT_Values.RA.addCentrifugeRecipe(GT_ModHandler.getModItem("GalaxySpace", "item.GlowstoneDusts", 1L, 3), GT_Values.NI, GT_Values.NF, GT_Values.NF, new ItemStack(Items.glowstone_dust, 2, 0), CustomItemList.ProteusStoneDust.get(1L), CustomItemList.ProteusStoneDust.get(1L), GT_Values.NI, GT_Values.NI, GT_Values.NI, new int[]{10000, 9000, 7500}, 300, 120);
 
-        GT_Values.RA.addCentrifugeRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.QuartzSand, 1L), GT_Values.NI, GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Stone, 1L), GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.CertusQuartz, 1L), GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.NetherQuartz, 1L), GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Quartzite, 1L), GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Cassiterite, 1L), GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.SiliconDioxide, 1L), new int[]{4000, 3000, 2000, 1500, 1000, 750}, 600, 16);
+        GT_Values.RA.addCentrifugeRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.QuartzSand, 9L), GT_Values.NI, GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Stone, 1L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.CertusQuartz, 1L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.NetherQuartz, 1L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Quartzite, 1L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Cassiterite, 1L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 1L), new int[]{9000, 3000, 2000, 1500, 1000, 750}, 5400, 16);
```

Comment:
i would lower the processing time. Sure it was more expensive before but i lower it for stone dust production as well

Reply:
maybe use 2400

Reply:
compared to stone dust use 108 sec = 2160